frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1418,12 +1426,22 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
             },
         ],
         replayFilters: [
-            (s) => [s.webAnalyticsFilters, s.dateFilter, s.shouldFilterTestAccounts],
+            (s) => [s.webAnalyticsFilters, s.dateFilter, s.shouldFilterTestAccounts, s.conversionGoal],
             (
                 webAnalyticsFilters: WebAnalyticsPropertyFilters,
                 dateFilter,
-                shouldFilterTestAccounts
+                shouldFilterTestAccounts,
+                conversionGoal
             ): RecordingUniversalFilters => {
+                const filters: UniversalFiltersGroupValue[] = [...webAnalyticsFilters]
+                if (conversionGoal) {
+                    const actionId = (conversionGoal as ActionConversionGoal).actionId
+                    const customEventName = (conversionGoal as CustomEventConversionGoal).customEventName
+                    const id = actionId || customEventName
+
+                    filters.push({ id, name: String(id), type: customEventName ? 'events' : 'actions' })
+                }
```

Comment:
@pauldambra Pinging you to learn if querying replay sessions this way makes sense or if I won't get the results I'm expecting. This `filters` entry is parsed into either the `actions` or `events` parameter that's sent to the backend.

I know `name` should be the name of the action rather than its id, but we don't have access to that very easily in this part of the app. Is it ok if `name` is not really the name? I assume it's fine and this is used for displaying purposes only - which we don't need.

Reply:
I don't actually know 🙈 

```
action = Action.objects.create(
            team_id=team_id,
            name=name,
            steps_json=[
                {
                    "event": name,
                    "properties": properties,
                }
            ],
        )
```

when setting up backend tests for filtering we're creating actions like this which suggests we care about the name and not the id 😬

Reply:
Welp, that's not good. I'll dive into that part of the code to learn if this will work or not, thx!